spec/concepts/api/v1/lib/operation/filtering_spec.rb

```diff
@@ -106,9 +106,7 @@
     context 'when filter is not a hash' do
       let(:filtering_params) { 'not_a_hash' }
       let(:errors) { { filter: [I18n.t('errors.hash?')] } }
-      let(:error_localizations) { %w[errors.hash?] }
 
-      include_examples 'errors localizations'
```

Comment:
Why did you remove it?

Reply:
This test had checked locales wich stored in gems
I think we should not duplicate them